facebook/__init__.py

```diff
@@ -46,7 +46,7 @@
 FACEBOOK_GRAPH_URL = "https://graph.facebook.com/"
 FACEBOOK_WWW_URL = "https://www.facebook.com/"
 FACEBOOK_OAUTH_DIALOG_PATH = "dialog/oauth?"
-VALID_API_VERSIONS = ["2.8", "2.9", "2.10", "2.11", "2.12", "3.0", "3.1"]
+VALID_API_VERSIONS = ["2.8", "2.9", "2.10", "2.11", "2.12", "3.0", "3.1", "3.2"]
```

Comment:
This line is too long. You can use [black](https://github.com/ambv/black) to automatically re-format your code.

Reply:
* reformatted with `black -l 79` as I've seen in another commit message
* forced pushed with amended commit message in present tense.

Cheers!